scripts/exglobal_stage_ic.sh

```diff
@@ -16,9 +38,21 @@ error_message(){
 }
 
 ###############################################################
-# Start staging
+# Start staging gefs and gfs here
+
+# Check for command line argument to select gfs or gefs
+if [[ $# -ne 1 ]]; then
+  echo "Usage: ${0} [gfs | gefs]"
+  exit 1
+fi
+
 
+if [[ "${selection}" = "gfs" ]]; then
```

Comment:
This should not be necessary, and is leading to a lot of duplication. Instead of these big if-blocks, there should be a small if-block based off of the `$RUN` that creates the list of `member_dirs` as you have above if it is `GEFS`, and otherwise makes it a one-element array with `""` (empty string) as the only entry. Once you've done this, you should simply loop over that for everything else and not need different blocks for GFS and GEFS.

Reply:
Is this way we have to OCN staging for gfs and gefs
if [[ "${DO_OCN:-}" = "YES" && "${RUN}" = "gfs" ]]; then

member_dir=""
NMEM_ENS=0
code here
elif [[ "${DO_OCN:-}" = "YES" && "${RUN}" = "gefs" ]]; then
code here (member_dir is already provided and NMEM_ENS as well so no need to define here?)

Reply:
No, you should only have one check of `$RUN` to make the memdir list. Everything after that is just looping through that list. You do not care whether it is GFS or GEFS once you have that list.

```
for MEMDIR in ${memdir[@]}; do
   # All the transfer code
done
```

Reply:
It's also important to use `MEMDIR`, as that is the variable that appears in the COM templates (see `config.com`). Otherwise the destinations won't be correct.